src/addons/dragAndDrop/EventWrapper.js

```diff
@@ -119,25 +125,39 @@ class EventWrapper extends React.Component {
       }
 
       /*
-      * props.children is the singular <Event> component.
-      * BigCalendar positions the Event abolutely and we
-      * need the anchors to be part of that positioning.
-      * So we insert the anchors inside the Event's children
-      * rather than wrap the Event here as the latter approach
-      * would lose the positioning.
-      */
-      children = React.cloneElement(children, {
+  * props.children is the singular <Event> component.
+  * BigCalendar positions the Event abolutely and we
+  * need the anchors to be part of that positioning.
+  * So we insert the anchors inside the Event's children
+  * rather than wrap the Event here as the latter approach
+  * would lose the positioning.
+  */
+      const newProps = {
         onMouseDown: this.handleStartDragging,
         onTouchStart: this.handleStartDragging,
         // replace original event child with anchor-embellished child
+
         children: (
           <div className="rbc-addons-dnd-resizable">
             {StartAnchor}
             {children.props.children}
             {EndAnchor}
           </div>
         ),
-      })
+      }
+
+      if (
+        draggable.dragAndDropAction.interacting && // if an event is being dragged right now
+        isSameEvent(draggable.dragAndDropAction.event, event) // and it's the current event
```

Comment:
no need for the fancy helper, `===` should be fine here since we aren't editing this event

Reply:
I thought so too, but I tested and it only works until the user starts dragging, then the events are different.

Reply:
Hmm that's a bug somewhere then, let me take a look

Reply:
ok cool